fdbserver/workloads/MetaclusterManagementWorkload.actor.cpp

```diff
@@ -392,12 +392,16 @@ struct MetaclusterManagementWorkload : TestWorkload {
 		state bool exists = itr != self->createdTenants.end();
 		state bool hasCapacity = self->createdTenants.size() < self->totalTenantGroupCapacity;
 		state bool retried = false;
+		state bool preferAssignedCluster = deterministicRandom()->coinflip();
 
 		try {
+			state TenantMapEntry createEntry;
 			loop {
 				try {
-					Future<Void> createFuture =
-					    MetaclusterAPI::createTenant(self->managementDb, tenant, TenantMapEntry());
+					if (preferAssignedCluster) {
```

Comment:
To get more coverage of the expected case, we could do something like:

```suggestion
					if (preferAssignedCluster && (!retried || deterministicRandom()->coinflip())) {
```